ipa-core/src/protocol/basics/mod.rs

```diff
@@ -41,7 +42,7 @@ use crate::{
 /// difficulty of resolving `V` vs. `[V; 1]` issues for the known value type. `Reshare` hasn't been
 /// attempted.)
 pub trait BasicProtocols<C: Context, V: SharedValue + Vectorizable<N>, const N: usize = 1>:
-    SecretSharing<V> + Reveal<C, Output = <V as Vectorizable<N>>::Array> + SecureMul<C> + FromPrss
+    SecretSharing<V> + Reveal<C, Output = <V as Vectorizable<N>>::Array> + SecureMul<C>
```

Comment:
@andyleiserson `FromPrss` bound cannot be fulfilled for MAC-malicious shares, so I had to remove it. Does not seem that it breaks anything, but let me know if it may stay in your way of adding some other functionality